test/Reader/Entry/_files/Atom/content/plain/atom10_Xhtml_alt.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+    <entry>
+		<content xmlns:xhtml="http://www.w3.org/1999/xhtml" type="xhtml">
+			<xhtml:div xmlns:xhtml="http://www.w3.org/1999/xhtml">XXX</xhtml:div>
+		</content>
+    </entry>
+</feed>
```

Comment:
For consistency with other unit test I would suggest the following file content:

```xml
<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
    <entry>
        <content type="xhtml" xmlns:x="http://www.w3.org/1999/xhtml">
            <x:div>Entry Content &amp;x:</x:div>
        </content>
    </entry>
</feed>
```